deepspeed/runtime/engine.py

```diff
@@ -3608,6 +3608,9 @@ def compile(self, backend=get_accelerator().get_compile_backend(), compile_kwarg
         """Compile the module using the specified backend and kwargs.
         If a compiler_fn is set, it will be used instead of torch.compile().
         """
+        # Avoid graph breaks
+        deepspeed.utils.nvtx.enable_nvtx = False
```

Comment:
@tohtana Do you want to set this based on the config - "compiler enabled =True"?

Reply:
We don't have the config any more